compat doc for bug#563497 (#977)
Microsoft/dotnet: Documentation/compatibility/aspnet-Invalid results from HttpRequest.GetAttributeFromHeader.md

> ## ASP.NET System.Web.HttpRequest.GetAttributeFromHeader can return invalid results

Reviewer: A problem, though, is that GetAttributeFromHeader is a private method. It would be useful to list some of the calling methods in the public surface area.

Author: see the latest update

### Version Introduced
4.8

### Change Description
When parsing an HTTP header value using `GetAttributeFromHeader`, ASP.Net might return a value that includes
a trailing delimiter prior to 4.8. Applications that target 4.8 or later with `targetFrameworkVersion` will now
have all delimiters stripped from return values.

### Recommended Action

> Starting from .NET 4.8, when targeting 4.8 through `targetFrameworkVersion`, the default behavior changes to strip

Reviewer: ```suggestion
Starting with applications running on .NET Framework 4.8, when targeting .NET Framework 4.8 or later by using the `targetFrameworkVersion` element, the default behavior changes to strip
```

> delimiters. When targeting previous framework versions, or not using `targetFrameworkVersion`, trailing delimiters for

Reviewer: ```suggestion
delimiters. When targeting previous .NET Framework versions or not using a `targetFrameworkVersion`, trailing delimiters for
```

Author: will update

> some values will still be returned. Alternatively, this behavior can be explicitly controlled with an `appSetting`:

Reviewer: ```suggestion
some values are still returned. 

This behavior can also be explicitly controlled with an `appSetting`:
```
